src/components/SkillsItem/index.jsx

```diff
@@ -0,0 +1,18 @@
+
+function SkillsItem({ skills }) {
+
+    return (
+        <>
+            {skills.map(skill =>
```

Comment:
You'll need a key prop in here, I'm getting a console error at the moment on this section:

Each child in a list should have a unique "key" prop.